File: src/java/katalet.java
class Katalet
{
    public static void main (String[] args)
    {
        if ( args.length  == 0 ) {
            System.out.format("Please enter a katalet name.\n");
            System.exit(1);
        }

        String kataName = args[0];
        System.out.format("The katalet you have selected is %s\n", kataName);
        System.exit(0);
    }
}

File: src/python/katalet.py
import sys

def main(argv):
    try:
        kataName = argv[1]
    except:
        print 'Please enter a katalet name.'
        exit(2)

    print 'The katalet you have selected is', kataName

if __name__ == "__main__":
        main(sys.argv[1:])
